build-scripts/babel.js

```diff
@@ -4,6 +4,7 @@ module.exports.babelLoaderConfig = ({ latestBuild }) => {
   }
   return {
     test: /\.m?js$|\.tsx?$/,
+    exclude: [require.resolve("@mdi/js/mdi.js"), require.resolve("hls.js")],
```

Comment:
These files are already ES5 and so do not need to be transpiled. These are the two generating a warning in Babel.

Note, most of our node modules are actually ES5. Mainly our Polymer stuff is not. We could switch to a whitelist of transpilation with the occasional breaking of the ES5 build because we shipped an arrow function.